tfx/src/main/scala/com/spotify/noether/tfx/TfmaConverter.scala

```diff
@@ -29,33 +30,80 @@ trait TfmaConverter[A, B, T <: Aggregator[A, B, _]] {
 
 object TfmaConverter {
 
+  private def denseMatrixToMetric(threshold: Option[Double] = None)(
+    matrix: DenseMatrix[Long]): MetricsForSlice = {
+    val tp = matrix.valueAt(1, 1).toDouble
+    val tn = matrix.valueAt(0, 0).toDouble
+    val fp = matrix.valueAt(1, 0).toDouble
+    val fn = matrix.valueAt(0, 1).toDouble
+
+    val cmBuilder = ConfusionMatrixAtThreshold
+      .newBuilder()
+      .setFalseNegatives(fn)
+      .setFalsePositives(fp)
+      .setTrueNegatives(tn)
+      .setTruePositives(tp)
+      .setPrecision(tp / (tp + fp))
+      .setRecall(tp / (tp + fn))
+
+    threshold.foreach(cmBuilder.setThreshold(_))
+
+    MetricsForSlice
+      .newBuilder()
+      .setSliceKey(SliceKey.getDefaultInstance)
+      .putMetrics(
+        "Noether_ConfusionMatrix",
+        MetricValue
+          .newBuilder()
+          .setConfusionMatrixAtThresholds(
+            ConfusionMatrixAtThresholds
+              .newBuilder()
+              .addMatrices(cmBuilder.build())
+              .build())
+          .build()
+      )
+      .build()
+  }
+
+  implicit val errorRateSummaryConverter
+    : TfmaConverter[Prediction[Int, List[Double]], (Double, Long), ErrorRateSummary.type] =
+    new TfmaConverter[Prediction[Int, List[Double]], (Double, Long), ErrorRateSummary.type] {
+      override def convertToTfmaProto(underlying: ErrorRateSummary.type)
+        : Aggregator[Prediction[Int, List[Double]], (Double, Long), MetricsForSlice] =
+        ErrorRateSummary.andThenPresent { ers =>
+          MetricsForSlice
+            .newBuilder()
+            .setSliceKey(SliceKey.getDefaultInstance)
+            .putMetrics(
+              "Noether_ErrorRateSummary",
+              MetricValue
+                .newBuilder()
+                .setBoundedValue(
+                  BoundedValue
```

Comment:
What's the difference between `BoundedValue` and `DoubleValue`?

Reply:
Ah, that's my bad. Mistakenly thought that you had to wrap DoubleValue in a BoundedValue